BSM/Chapter_04_01_Logistic_regression_for_NBA_clutch_free_throws.ipynb

```diff
@@ -0,0 +1,426 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pymc3 as pm\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import arviz as az"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "az.style.use('arviz-white')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Logistic regression for NBA clutch free throws\n",
+    "\n",
+    "### Chapter 4.3.3: Generalized linear models\n",
+    "\n",
+    "The NBA clutch free throws data set has three variables for player $i=1,...,10$:\n",
+    "\n",
+    "1.  $Y_i$ is the number clutch free throws made\n",
+    "2.  $N_i$ is the number clutch free throws attempted\n",
+    "3.  $q_i$ is the proportion of the non-clutch free throws made\n",
+    "\n",
+    "We model these data as\n",
+    "$$Y_i\\sim\\mbox{Binomial}(N_i,p_i),$$\n",
```

Comment:
The binom is not showing in Github. Initially I was confused until I took a look at the source code
![image](https://user-images.githubusercontent.com/7213793/61096609-d4a37600-a41d-11e9-8030-956ca5275dfd.png)